### DISCUSSION TITLE
Perplexed about "Inbox" - can't seem to work out how to get it the feature, basically

### DISCUSSION THREAD

**Discussion by ajkavanagh** [jonaswinkler/paperless-ng#1010] [Support] [answered]
Hi

So I managed to get a installation of paperless ng-1.4.2 going in LXD containers using ansible roles (it's reproducible if anyone would like to also go that route).  I used LXD simply because that's what I'm running on the server.

I'm really interested in paperless-ng, but I'd also really like an 'Inbox' for newly added documents, and it "looks" like that feature does exist (as I've seen it in the various docs and pictures).  My setup is:

- paperless-ng (1.4.2).
- shared drive with syncthing so I can drop files into the consumption folder.

Paperless sees the file, consumes it and then just silently adds it to the collection.  There isn't an "Inbox" I can find that shows me documents that haven't been manually processed.

Is there such a feature?  I've attached my current view of the set up below.

Thanks,
Alex.

![image](https://user-images.githubusercontent.com/567675/118108613-1b76a400-b3d8-11eb-8795-217467a23f00.png)

**Comment by jonaswinkler** [comment, marked as answer]
Add a new tag and tick the "Inbox tag" checkbox.

Then create a saved view that shows all documents with this tag.

**Reply by Burschi500** [reply]
> Then create a saved view that shows all documents with this tag.

... As admin from the admin panel. AFAIK not possible as normal user.

**Reply by ajkavanagh** [reply]
> Add a new tag and tick the "Inbox tag" checkbox.
> 
> Then create a saved view that shows all documents with this tag.

Awesome!  Thanks very much.  I must have missed that the first time around.
